manifests/vhost.pp

```diff
@@ -2707,12 +2711,27 @@
   # - $ssl_openssl_conf_cmd
   # - $ssl_stapling
   # - $apache_version
-  if $ssl {
+  if $ssl and $ensure == 'present' {
```

Comment:
Should it actually ensure the old certificate copies are absent if `$ensure` is `absent`?

Reply:
Not needed, the puppet_ssl_dir is managed with purge => true, so old copies will be cleaned up by that.